cmd/csi-node-driver-registrar/node_register.go

```diff
@@ -74,6 +80,41 @@ func nodeRegister(
 		klog.Errorf("Registration Server stopped serving: %v", err)
 		os.Exit(1)
 	}
+
 	// If gRPC server is gracefully shutdown, exit
 	os.Exit(0)
 }
+
+func periodicallyCheck(socketPath string, registrar registerapi.RegistrationServer) {
```

Comment:
function name: what are you periodically checking?

Reply:
is there a suitable name for the function? @jingxu97